rest/types/cluster.go

```diff
@@ -28,6 +28,17 @@ type ClusterMemberLocal struct {
 // MemberStatus represents the online status of a cluster member.
 type MemberStatus string
 
+// RoleStatus represents the previous and current role of a cluster member.
+type RoleStatus struct {
+	OldRole string
```

Comment:
Not a blocker bot I am wondering if we should duplicate `Role` in the names too as the struct itself already indicates we are talking about roles here?

Reply:
I agree that it's unnecessary in the field names. Fixed!